apps/backend/src/libs/modules/http/libs/exceptions/exceptions.ts

```diff
@@ -1 +1 @@
-export { HTTPError } from "shared";
+export { AuthError, HTTPError } from "shared";
```

Comment:
AuthError should be exported from ~/modules/auth/auth.ts, not http

Reply:
Maybe we can do without exporting this error from auth module since Auth error is used only inside of AuthController?

Reply:
It is a part of auth module, so it should be exported from there